source/guide_kirby.rst

```diff
@@ -0,0 +1,161 @@
+.. sectionauthor:: Daniel Kratz <uberlab@danielkratz.com>
+.. highlight:: console
+
+.. sidebar:: Logo
+  
+  .. image:: _static/images/kirby.svg 
+      :align: center
+
+#########
+Kirby
+#########
+
+Kirby_ is a modern, fast, flexible, file-based (no database) content management system written in PHP. 
+
+The idea behind Kirby is to provide a tool for developing websites or applications that is both easy to use for developers and at the same time gives you all the freedom you want to structure your content and render your output.
+
+Kirby was released for the first time in 2009 and is actively developed by the Bastian Allgeier GmbH. Although the code is open source, a license is required for use in production. 
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * PHP_
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1: 
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+The installation of Kirby is quite easy. Just clone the Kirby Starterkit in your `document root`_. 
+
+.. code-block:: console
+ :emphasize-lines: 2
+
+ [isabell@stardust ~]$ cd ~/html
+ [isabell@stardust html]$ git clone --recursive https://github.com/getkirby/starterkit.git ~/html
+  Cloning into 'html'...
+  remote: Counting objects: 3749, done.
+  remote: Compressing objects: 100% (69/69), done.
+  remote: Total 3749 (delta 25), reused 92 (delta 25), pack-reused 3649
+  Receiving objects: 100% (3749/3749), 7.74 MiB | 9.97 MiB/s, done.
+  Resolving deltas: 100% (1035/1035), done.
+ [isabell@stardust html]$
+
+That's all the magic. If you visit your previously set up domain you should see a working website with demo contents from the Starterkit. 
+
+.. note:: While we are cloning Kirby from Github you can alternatively just download_ the zip and upload the contents manually to your `document root`_.
+
+To simplify future updates we are replacing the ``kirby`` and ``panel`` folders and add them again as Git submodules. Using this workflow, updates will need only two commands on the command line. 
+
+First, we remove the existing folders from the filesystem. Then we stage the changes in Git to remove the folders from the repository index. 
+
+.. code-block:: console
+ :emphasize-lines: 2,3
+
+ [isabell@stardust ~]$ cd ~/html
+ [isabell@stardust html]$ rm -R panel kirby
+ [isabell@stardust html]$ git add .
```

Comment:
It seems like the starterkit always contains the latest version anyway. What is the advantage of replacing these directories instead of calling `git pull` on the starterkit directly to update in the future?

Reply:
(Sorry, didn't see your second comment.) I see the problem if I change the origin to my own repo in order to version control my project. As soon as I do this, git pull would not get updates from the old origin, or do I miss something?

Reply:
So I found the [corresponding commit](https://github.com/getkirby/starterkit/commit/40cb82e84a5d1520b84db09c72430265af50e469). Other than stated in the documentation, Submodules are apparently not the preffered way anymore. The mentioned Kirby CLI is comparable to WP CLI. Since the WordPress guide is also completly dependent on the CLI I think I will also rework this guide to use the Kirby CLI for installation and updates. Thanks for your input on this!